examples/src/main/java/com/influxdb/v3/ProxyExample.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.influxdb.v3;
+
+import java.util.stream.Stream;
+
+import com.influxdb.v3.client.InfluxDBClient;
+import com.influxdb.v3.client.Point;
+import com.influxdb.v3.client.PointValues;
+import com.influxdb.v3.client.config.ClientConfig;
+
+public final class ProxyExample {
+
+    private ProxyExample() { }
+
+    public static void main(final String[] args) throws Exception {
+        // Run docker-compose.yml file to start Envoy proxy
+
+        String proxyAddress = new InetSocketAddress("localhost", 10000);
+        String certificateFilePath = "src/test/java/com/influxdb/v3/client/testdata/influxdb-certificate.pem";
+        ClientConfig clientConfig = new ClientConfig.Builder()
+                .host(System.getenv("TESTING_INFLUXDB_URL"))
+                .token(System.getenv("TESTING_INFLUXDB_TOKEN").toCharArray())
+                .database(System.getenv("TESTING_INFLUXDB_DATABASE"))
+                .proxyAddress(proxyAddress)
```

Comment:
So should `proxyAddress` be a `String` or `InetSocketAddress`?  So that users don't have to use too many imports, I'd prefer using more primitive or _core_ classes in the configuration as much as possible.  Passing these values to other libraries should also be handled internally as much as possible. 

![Screenshot from 2025-03-20 14-23-17](https://github.com/user-attachments/assets/d1c953a4-5458-4700-9326-d9b82803e059)